docs/README.md

```diff
@@ -0,0 +1,217 @@
+    - [2. Fetching test session data using Kobiton REST API](#2-fetching-test-session-data-using-kobiton-rest-api)
+- [C. Feedback](#c-feedback)
+
+## A. Integrating Kobiton with CircleCI
+### 1. Preparation
+#### 1.1. Getting Kobiton Username and API key
+Kobiton Username and API key are required for authenticating with Kobiton.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+#### 1.2. Samples
+In this repository, we have already provided samples for executing automation test in Kobiton:
+- Script for executing automation test on Kobiton devices (`/samples/automation-test/automation-test-script.js`).
+- CircleCI configuration file (`/.circleci/config.yml`).
+
+Link to the `samples` folder: https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative/tree/master/samples
+
+In this guide, we will be using the provided samples. Follow steps below to get started:
+
+1. Fork this repository https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative
+2. Synchronize the forked repository with CircleCI.
+> Refer to https://circleci.com/docs/2.0/getting-started/ for instructions on how to synchronize a repository with CircleCI.
+> Note : At first synchronization, the build process will be failed to execute. This is normal, just ignore the `Build Failed` message.
+
+### 2. Setup
+#### 2.1. Setting Kobiton Username and API key
+
+In your project configuration in CircleCI, add two environment variables :
+- KOBITON_USERNAME - Your Kobiton's username.
+- KOBITON_API_KEY - Your Kobiton's API Key.
+
+Your `Environment Variables` window should look like this
+
+![Environment variables](./assets/env-variables-after-add.png)
+
+#### 2.2. Setting Kobiton device desired capabilities
+In order to execute tests with a specific device in Kobiton, its corresponding desired capabilities needs to be supplied.
+
+In `samples/automation-test` folder, we have provided a sample script for executing automation test on Kobiton device(s). The provided script has been pre-configured to execute automation test of a demo Android application on one of available devices which name begins with `Galaxy`.
+
+If you don't want to use default configuration, follow these steps below to get and configure desired capabilities for a specific device.
+
+**Getting desired capabilities**
+1. Go to https://portal.kobiton.com/login and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run the test script on (in this example: `Pixel 2 XL`), click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
```

Comment:
Need to update the image due to the latest information (groupID, related messages, etc.)